### ISSUE TITLE
REPEATED_FIELD_NAMES_PLURALIZED should account for uncountable names

### ISSUE THREAD

**Issue by CoRfr** [yoheimuta/protolint#447]
While adding a field
```
repeated AdditionalInfo additional_info = 6;
```
protolint raised the following issue:
```
Repeated field name "additional_info" must be pluralized name "additional_infos"
```

My understanding is that `info` is uncountable, and therefore that the pluralized version of `info` is also `info`.

**Comment by yoheimuta** [comment]
That's right. Thank you for letting me know.

**Comment by yoheimuta** [comment]
📝  You can implement a workaround to adjust uncountable names by using a config file, such as https://github.com/yoheimuta/protolint/blob/71e65031e935e1c7924ea30911f87b6e8cd9ebfe/_example/config/.protolint.yaml#L118-L120.

**Comment by jduan-highnote** [comment]
Our CI job picked up the latest release (0.53.0) and now it's failing due to the following:

```
[protos/proto-onboarding/src/main/java/proto/onboarding/onboarding.proto:413:3] Repeated field name "preliminary_business_infos" must be pluralized name "preliminary_business_info"
```

How do I allow "preliminary_business_infos" in .protolintrc? Thanks!

**Comment by yoheimuta** [comment]
@jduan-highnote You can add `plural_rules` like the below.

```yaml
    repeated_field_names_pluralized:
      plural_rules:
        "(?i)info$": "infos"
```

**Comment by jduan-highnote** [comment]
Tried our suggestion and it worked well! Thanks.
